draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -654,476 +526,437 @@
+	Error.v2 Description</xref>.
+   </t>
 
    <t>
-   The URL of dCDN's collection of all Trigger Status Resources
-   needs to be either discovered by or configured in uCDN. The
-   mechanism for discovery of that URL is outside the scope of this
-   document.</t>
+	The new trigger resource is created in "pending" state. 
+	If successful, The HTTP response to the uCDN trigger creation request MUST have status code 201 (Accepted) and MUST convey the 
+	URI of the newly created trigger resources in the Location response header field <xref target="RFC9110"/>. 
+	The HTTP response SHOULD include the updated representation of the trigger resource. 
+   	This is particularly important in cases where dCDN processed the trigger immediately.</t>
 
    <t>
-   CI/T Commands are POSTed to dCDN's collection of all Trigger
-   Status Resources. If a CI/T Trigger Command is accepted by dCDN,
-   dCDN creates a new Trigger Status Resource and returns its URI to
-   uCDN in an HTTP 201 response. The triggered activity can then be
-   monitored by uCDN using that resource and the collections
-   described in <xref target="collections-of-trigger-status-resources"/>.</t>
+	Once a trigger resource has been created, dCDN MUST NOT reuse its URI, even after the trigger resource has been fully removed. 
+	It it therefore recommended for dCDN to utilize unique UUID identifiers. 
+   </t>
 
    <t>
-   The URI of each Trigger Status Resource is returned to uCDN when
-   it is created, and URIs of all Trigger Status Resources are listed in
-   dCDN's collection of all Trigger Status Resources. This means
-   all Trigger Status Resources can be discovered by uCDN, so dCDNs
-   are free to assign whatever structure they desire to the URIs for
-   CI/T resources. Therefore, uCDNs MUST NOT make any assumptions
-   regarding the structure of CI/T URIs or the mapping between CI/T
-   objects and their associated URIs. The URIs used in the examples in
-   this document are purely illustrative and are not intended to impose
-   a definitive structure on CI/T interface implementations.</t>
-
-   <section title="Creating Triggers" anchor="creating-triggers"><t>
-   To issue a CI/T Command, uCDN makes an HTTP POST to dCDN's
-   collection of all of uCDN's Trigger Status Resources. The
-   request body of that POST is a CI/T Command, as described in
-   <xref target="cit-trigger-create-command"/>.</t>
+	dCDN SHOULD provide continuous update of the trigger processing progress, by responding with updated trigger resource representation 
+	to subsequent uCDN requests sent to the created trigger URL. 
+	If the dCDN is unable to do that, it MUST indicate that it has accepted the request but will not be providing further status updates. 
+	To do this, it MUST set the trigger state to "processed" at creation time. 
+	In this case, CI/T processing should continue as for a "complete" request.
```

Comment:
This language was original, but it also sounds awkward.  Maybe, "CI/T process should continue until it reaches a "complete" state."

Reply:
fixed